functions/src/modules/SE.functions.ts

```diff
@@ -16,15 +16,13 @@ export const uploadProcessing = functions.storage.object()
 
     const filePath = object.name;
     
-    if (object.name.slice(-4).toLowerCase() !== 'flac') {
-        console.error("Something is wrong with the file extension!")
-        return -1;
-    }
-
     const filePathRegex = /sound-editing\/restored\/(\w+)\/(\w+)/g;
 
-    if (!filePath.match(filePathRegex)) {
-        console.error("Something is wrong with the file path!")
+    if (!filePath.match(filePathRegex)) 
+        return -1;    
```

Comment:
Just check for the path prefix here to filter out the events. Regex should be more specific and should used after this simple check.